### ISSUE TITLE
HLS audio only live stream, Audio stops playing after first set of chunks and then buffering indefinitely 

### ISSUE THREAD

**Issue by asheeshs** [google/ExoPlayer#4489]
### Issue description
When played an HLS audio only live stream (will email the link), audio stops playing after few seconds and  then ExoPlayer goes into buffering state indefinitely with continuous callbacks to onTimelineChanged.
Also seeing a lot of "AudioTrack: Discontinuity detected [expected 6452760, got 528678]" messages in the logs.

### Reproduction steps

- In ExoPlayer demo app  (https://github.com/google/ExoPlayer/tree/release-v2/demos)
- Add below HLS stream in media.exolist.json
     {
        "name": "WNYC HLS stream - WNYC.org",
        "uri": "will email the link"
      }
- Build and launch the app
- Play the link
- Notice that Audio plays for a few seconds and then there is silence.
- Log shows player state changes to buffering and many "onTimelineChanged" and AudioTrack: Discontinuity detected".

### Link to test content
will email

### Version of ExoPlayer being used
Tried it with both 2.6.1, 2.8.2

### Device(s) and version(s) of Android being used
- Pixel2 running Android 8.1.0
- Samsung S7 running Android 8.0.0
- I think it should happen on any Android device.

### A full bug report captured from the device
Will email

**Comment by AquilesCanta** [comment]
I think the ID3 timestamps at the start of each segment are wrong. I got the first few:

2293397053
2293403636
2293409635
2293415635
2293421635
2293427635
2293433635
2293439635
2293445635
2293451635

which in seconds (the value is supposed to be a TS-PTS, a 90kHz clock) is:

25482.18
25482.26
25482.32
25482.39
25482.46
25482.52
25482.59
25482.66
25482.72
25482.79

The timestamps advance but not at a pace of ~6 seconds, which is the declared duration. There is probably a bug in the _timestamper_. Hardcoding HlsMediaChunk to ignore the ID3 timestamp seems to make things work. However, the solution is fixing the ID3 timestamp, which is mandatory according to the spec. Do let me know if I got anything wrong and we will reopen.
